### PR TITLE
Project Assignment

### PR DESCRIPTION
**Pull request by TinySquid**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -1,51 +1,108 @@
...
 # If the user enters "q", quit the game.
+
+
+def print_guide():
+    print("\nMovement commands: [n]orth, [s]outh, [e]ast, [w]est")
+    print("Other commands: [i]nventory, [p]ickup, [d]rop, [q]uit\n")
+
+
+# Clear screen in preparation for game
+os.system("cls")
```
Comment (jdorety) [inline src/adv.py]: This isn't breaking anything, but my it is making my shell throw an error every time it's called

File: src/adv.py [hunk 2]
```diff
@@ -1,51 +1,108 @@
...
 
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player("Bob Ross", room["outside"])
+
+# Possible player commands
+verb_cmds = {
```
Comment (jdorety) [inline src/adv.py]: nice lambda functions

File: src/adv.py [hunk 3]
```diff
@@ -1,51 +1,108 @@
...
+        "The narrow passage bends here from west to north. The smell of gold permeates the air.",
+    ),
+    "treasure": Room(
+        "Treasure Chamber",
+        "You've found the long-lost treasure chamber! Sadly, it has already been completely emptied by earlier adventurers. The only exit is to the south.",
+    ),
 }
 
 
 # Link rooms together
```
Comment (jdorety) [inline src/adv.py]: Think of how you might implement the room class in a way where you wouldn't need to alter these calls.

File: src/player.py [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+
+    def __init__(self, name, initial_room):
+        # Player info
+        self.name = name
+        self.current_room = initial_room
+
+        # Player inventory
+        self.inventory = []
+
+        self.directions = {"n": "north", "s": "south", "e": "east", "w": "west"}
```
Comment (jdorety) [inline src/player.py]: Is this necessary?  Where is it being referenced?

File: src/adv.py (hunk 1 above)
Comment (TinySquid) [reply src/adv.py]: Fixed in a later commit

File: src/adv.py (hunk 3 above)
Comment (TinySquid) [reply src/adv.py]: Changed back to doc strings in a later commit

File: src/player.py (hunk 4 above)
Comment (TinySquid) [reply src/player.py]: Referenced on line 31, basically just converts the n, s, e, w into "north", "south" etc for printing to console

**Pull request merged**